clients/fides-js/src/lib/overlay.module.css

```
@@ -26,10 +27,13 @@
   );
   /* Text */
   --fides-consent-overlay-title-font-color: var(
-    --fides-consent-overlay-font-color
+    --ides-consent-overlay-font-color
```

whoops! looks like there are a few other instances too

```suggestion
    --fides-consent-overlay-font-color
```